packages/components/src/mobile-promo-card/index.tsx

```diff
@@ -0,0 +1,112 @@
+import classNames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+import './style.scss';
+import iconWoo from './images/icon-woo.png';
+import qrCodeJetpack from './images/qr-code-jetpack.png';
+import qrCodeWoo from './images/qr-code-woo.png';
+import storeBadgeApple from './images/store-apple.png';
+import storeBadgeGoogle from './images/store-google.png';
+import { WordPressJetpackSVG } from './svg-icons';
+
+export type MobilePromoCardProps = {
+	className?: string;
+	isWoo?: boolean;
+};
+
+export default function MobilePromoCard( { className, isWoo }: MobilePromoCardProps ) {
+	// Using useTranslate() with interpolation to set up the title/message.
+	// https://wpcalypso.wordpress.com/devdocs/packages/i18n-calypso/README.md
+	const translate = useTranslate();
+	const redirectLink = isWoo ? 'https://woo.com/mobile/' : 'https://jetpack.com/app/';
+	const linkClassName = isWoo ? 'woo' : 'jetpack';
+	const components = {
+		a: <a className={ linkClassName } href={ redirectLink } />,
+	};
+	// Basic user agent testing so we can show app store badges on moble.
+	const userAgent = window.navigator.userAgent.toLowerCase();
+	const isApple = userAgent.includes( 'iphone' ) || userAgent.includes( 'ipad' );
+	const isGoogle = userAgent.includes( 'android' );
+
+	const getTitle = () => {
+		if ( isWoo ) {
+			return translate( 'Bring your Store stats with you using the {{a}}Woo{{/a}} mobile app', {
+				components,
+			} );
+		}
+		return translate( 'Bring your stats with you using the {{a}}Jetpack{{/a}} mobile app', {
+			components,
+		} );
+	};
+
+	// Determines message text based on mobile, tablet, or Desktop.
+	const getMessage = () => {
+		if ( isApple ) {
+			return translate(
+				'Check your stats on-the-go and get real-time notifications with the Jetpack mobile app.'
+			);
+		} else if ( isGoogle ) {
```

Comment:
```suggestion
		}
		
		if ( isGoogle ) {
```

This `else` could be ignored, which will not be a blocker.

Reply:
I'll have another PR up soon with these changes. Thanks, as always, @dognose!